templates/riscv/qemu-sifive-u/mods.config

```diff
@@ -17,16 +18,40 @@ configuration conf {
 	/* Tell printf() do not support floating point */
 	include embox.compat.libc.stdio.print(support_floating=0)
 
+	@Runlevel(0) include embox.mem.phymem
+	@Runlevel(1) include embox.kernel.timer.sys_timer
+	@Runlevel(1) include embox.kernel.time.kernel_time
+
+	@Runlevel(2) include embox.kernel.irq
+	@Runlevel(2) include embox.kernel.critical
+	@Runlevel(2) include embox.kernel.timer.sleep
+	@Runlevel(2) include embox.kernel.timer.strategy.list_timer
+	@Runlevel(2) include embox.kernel.time.timekeeper
+	@Runlevel(2) include embox.kernel.task.multi
+	@Runlevel(2) include embox.kernel.thread.core
+	include embox.kernel.stack
+	include embox.kernel.sched.strategy.priority_based
+	include embox.kernel.thread.signal.sigstate
+	include embox.kernel.thread.signal.siginfoq
+
+
 	include embox.kernel.critical
 	include embox.compat.libc.math_builtins
 	include embox.kernel.spinlock(spin_debug = false)
 
+	include embox.kernel.task.resource.vfork
+	include embox.compat.posix.proc.exec
+	include embox.compat.posix.proc.pid
+	include embox.compat.posix.proc.waitpid
+	include embox.compat.posix.fs.close
+
 	include embox.kernel.task.resource.errno
 
 	include embox.compat.libc.str
+	include embox.framework.LibFramework
 
 	include embox.test.stdlib.setjmp_test
 	include embox.test.kernel.timer_test
-	include embox.test.kernel.thread.thread_test
```

Comment:
Why you deleted thread_test from config? Does it work normally?

Reply:
Yes, it does. Brought it back.

Reply:
I got the following:

```
Embox kernel start
	unit: initializing embox.mem.phymem: 
			start=0x80069680 end=0xa0000000 size=536439168
		done
runlevel: init level is 0
	unit: initializing embox.kernel.task.task_resource: done
	unit: initializing embox.kernel.task.task_table: done
	unit: initializing embox.kernel.task.kernel_task: done
	unit: initializing embox.driver.interrupt.riscv_intc: done
	unit: initializing embox.kernel.sched.sched: done
	unit: initializing embox.arch.riscv.kernel.interrupt: done
	unit: initializing embox.driver.clock.riscv_clk: done
	unit: initializing embox.kernel.time.timer_handler: done
	unit: initializing embox.kernel.time.kernel_time: done
runlevel: init level is 1
	test: running embox.test.hal.context_switch_test . done
	test: running embox.test.posix.vfork_test .... done
	test: running embox.test.kernel.thread.thread_test .

  ______
 |  ____|                                            __          __
 | |___  _ __ ___            ____  ____  ____  _____/ /   _____ / /
 |  ___|| '_ ` _ \          / __ \/ __ \/ __ \/ ___/ /   |_____| |
 | |____| | | | | |_ _ _   / /_/ / /_/ / /_/ (__  )_/    |_____| |
 |______|_| |_| |_(_|_|_)  \____/\____/ .___/____(_)           | |
                                     /_/                        \_\
 ASSERTION FAILED on CPU 0
	at src/kernel/thread/core.c:272
	in function thread_delete

!t->schedee.ready
```

Reply:
Launched like `qemu-system-riscv32 -M sifive_u -kernel ./build/base/bin/embox -m 512 -nographic`.

Reply:
Can you please check that?

Reply:
I launch qemu in thus way: `qemu-system-riscv32 -nographic -machine sifive_u -kernel build/base/bin/embox`
qemu 4.1.0

Reply:
Ok, thank you, I have the same qemu 4.1.0.

Reply:
BTW we investigated that the problem is in runlevels..

Reply:
The problem was in runlevels. I fixed it.

Reply:
Nice! Finally, it works for me. @Deryugin If you want you can try too on qemu 4.1.0 (without auto_qemu)